wiki/FAQ/en.md

```diff
@@ -1 +1,37 @@
-# FAQ
\ No newline at end of file
+# FAQ
+
+This is a list of common questions that get asked.
+
+## What is this?
+
+Titanic! is a private server, that was designed to support osu! stable clients from 2008 to 2014 and only maps from that era (With a few exceptions). It's a community-driven project that is fully [open-source](https://github.com/osuTitanic), so feel free to look at the code and contribute to it.
+
+## How can I play?
+
+To get started you can take a look at the [[installation guide]] or just go to the [downloads page](https://osu.titanic.sh/download/) on our website.
+
+## How can I get beatmaps.
+
+You can download beatmaps via osu! Direct or on the [beatmaps page](https://osu.titanic.sh/beatmapsets/) on the website. Downloading maps from the offical osu! website works as well.
+
+## My game is feeling slow/laggy, how can I fix that?
+
+Many players sadly face this issue and most often it can be fixed by setting the RefreshRate value in the osu!.yourname.cfg file, to your monitors refresh rate. If you are still experiencing issues, please try to mess around with the graphics & fps settings, until it feels right!
+
+## Will RX/AP scores be submitted?
+
+Yes we support Relax and AutoPilot submittion. We currently do not have seperate leaderboards for these at the moment. They do give PP and score but not as much compared to vanillia to keep things fair.
```

Comment:
```suggestion
Yes, we do support Relax and Autopilot submission, with one noticeable difference to other servers: Ranks for Vanilla, Relax and Autopilot are merged together, since they do give PP and score, but not as much compared to Vanilla to keep things fair.
```